processing/src/main/java/io/druid/query/Queries.java

```diff
@@ -64,6 +68,40 @@ public String apply(AggregatorFactory input)
         );
         combinedAggNames.add(postAgg.getName());
```

Comment:
why not do the check for post-aggregators naming here, before adding it to combinedAggNames?